### ISSUE TITLE
Righteous Fire still using 3.15 values

### ISSUE THREAD

**Issue by OleBueker** [PathOfBuildingCommunity/PathOfBuilding#3422]
### Check version

- [X] I'm running the latest version of Path of Building and I've verified this by checking the [changelog](https://github.com/PathOfBuildingCommunity/PathOfBuilding/blob/master/CHANGELOG.md)

### Check for duplicates

- [X] I've checked for duplicate issues by using the search function of the [issue tracker](https://github.com/PathOfBuildingCommunity/PathOfBuilding/issues)

### What is the expected behaviour/value?

Patch notes say:
Righteous Fire now deals 39 Base Fire Damage per second at gem level 1 (previously 35.8), up to 2447.9 at gem level 20 (previously 1541.8). It also now deals 35% of Life and Energy Shield as Base Fire Damage per second (previously 20%).

So I would expect the numbers to give the new 3.16 values.

### What is the actual behaviour/value?

It is showing the 3.15 values (1541.8 and 20%).

### How to reproduce the issue

Add (Vaal) Righteous Fire as a skill.

### Build code

```shell
eNq9PGlz2sqyn8OvUFF1Xp1TBluzaCT52fcW4I3EJBhsZ_mSGmAwioVEJAEmp_zfX48kQGCWQUm9uMpBUu_d08to8Nl_X4auNhFB6PjeeREd60VNeF2_53hP58WH-6uyVfzvfwpnTR4NPvWrY8eVT_5TeHcWf9ZcMRHuedEGtIgHTyJ6nJMi3-HeiHvRQPheg__wg2u_d1786HuiqHW413Oi-VXX5WH4kQ_FefGzE3UHRY2HXeH1asv7n7rdsRs5YVTUhtzx2n73WUTXgT8egdBFbeKIacPvAWC90fzUus9wdrwsZ5D83VnT5TMRtCMeaSH8Oi9WwAD8Sdw4QH7C3THQ0YsnO2Ev-BB-7wdvj4To_X-DNQNx2e-LbuRMRC1wotqAe10FYQ-FbUifjFxHBAt4dGxuw7h5QxzpW8nf-xF3L5rt_YIcCukvnYwoYkQ_1jE1LdNE2_AgKAeAluVxAGbVBZ8pySeh60-eEwll8KbvhL53kPY51Kj5w47jrWpxIGpl8qQQU77v9vypt4Q8JmTrQgwE_9RP4rzFe844XKBhtg2pwT1e80OFdX7r9IUa5GVbDa4FCUMNUkrZFAHkwEhd2IMQ2lFmyRp0q8ji5wqkvdXPF-Jlaf5d9LKAW8Wre0sdiI53kMtCIrRDvIkfQV1SM-WSNzbIjlzcPZXQda-bEYHsIvzgBSIUwSSTxHexWEVJHaySPSViSzyJpb6GYR8jE-9CuBWiO7iG6trikVAL0gWUjXbbSQJn7YR3kt1gJpBfDWPdSgwdY8xMbOgWRtTcRWTVYoQc72R5oLmS9LvqE2Qe2zbBO1E-imhVLoTtY8O0rG1YgHCw-wHnMAMAwqUngqdZe-AIt7eKWKaMHNMdin3mgUIXk2WQKTvGzkDLIq0EHFVhs2Y0emyoYB0YCZcTHmZTEWK7NUrAs8psXzwCmg1A6AnVNu5wjGbg_5CtpXsYWiUY-uNMKWFI36l2Aq-kdXMwC50urK64J2-J3ri7kuvx1rKw6JIb_kQMIW3EDTZMEpl-Fm_NGVUXxpA1_cmuPOi6m1C2ur8SRbz7fOH3npStHDM5HKM9Ho0ggco4U8W7cgIwdehkug0L7Qf-BONTjY-Wa2xHMyjX4wqDsrEfeJ3Bjhr5NIg8mGaVmaxhKHMCk_rhOhemAq7M4mrsrsxA2NQJhmke2Sayt_d37WdnFS_iHVecavoLxrbo6WJrlWn6UxBtIGf-Zd9NVaAb_GU_wlUgvF-zdfq6EniWAdmev3vjQEa6Mo91jLdszk7iTRH5qT4c-UEU36xxtxvGJOveaBxpXryhMXTC7vfOuN-X2xZFYBHEOy6XV1eXtfv642UqRRYllM767o2HHTlvJ__L3Y8Esi3ipKd1fdflo1D0zot97oZA2-mljm5Dru9GSvAwq6c7HCrQUM-7gQNGVAGWA7iSyPFehwqk3IpQAgSvcFeoG-x-NhIyAkJVm6XJUwU63hZQgkxmfDUF490DNTeILp8pQS7aGyXoS1dUHFfWUjWrNWAhJGVYDR6KYuB0xpFitMXjnJIccpxRUjDbliquDjXAtEVSEiLtIVVg08qjFpTQnii77UL0BYS74rKP08k9fxZemiznifEsXmuhFkLGvBbDsDqDinQlzbu287QCALUqGAPtnujzsSvv342560QzmcIzd2-THWoYOLVw4E9lp5OQkYs7BKPf3iZPKm6UUpA85rInmkkB443nStwtxtex-PHus-N13XFP1L20Cs9Fc3lHsm5dFbUnuVVd88ceFBLPceUmu6y0vTlo6PqyofR7M23eKK-xAxViYd6dgXgp-rXrd7iLF_yWqmafowUTSacOPB85d1uymRH-OJS9WVH7mahejzNDbDsQWgzldUNEvMcjflKPwDAn0jonsUjwaTul2A1rWnYTA4DJZEWTbX4ii7YgocU0TrYqmgaUsqapu2GMCASHkKyOA9nAzXfvD9V6ETx7CKoqn2Jr2Vr7J_WGZCwzMXev_K7cH82t7xZCqnou0LUE_88r2u_DBNid_Y6K6ySUlZsj_nGt4lEW5tF4py23YpuoqOpW98KRE_BoUWv-pHo135NbdUBe9JIW4De03EFMVdksCS2lkZS2JN_lqwVz4yTFoAIzRLi_HrxJ_qj4x6z_6ESxWTYbOwZr8iAxjkoJWCennPkXiDHPhuPFL47b4-HQh5LfEnwkgj8fdVWXh6OBGP62_ovQe0tROf8vMP-cmlcuHw55x9nuYgWnbiSiqtUK8toCSj_Ffd99IITGkyiPEVHabsFF9giAFBbJNRFGIpilw8J58V9qYWaWCCOIvJb-Zci0jBKyENHhClvYKiFkYCQfEcysEqbIko8ItbFZYhgxuAAQ3SgZBtEtuKLEQlaJWNSgMaCJSYnqRDflM0SRUSIUUyyvTGzTEsIGegXVYYYHgUAOSgEe26hkUvkRWQDDTBM4wDUwBQngtkUYLlFm2Qw4GwZwBB64RIhpAT9qg1aM2CYrYRPrBLBAOuBMQRps2KZeMnQLKNiYIRDBLFFsMr1EkE0QoDDDLsXqY8PQ4bllAbLNKNzQLYJBcwOEwEgHsahlYbiTmA4kB3xkg0CMUh2VDGQClmVTaUqKJTGDMPkQM1AODIcMsD5QQWA20yYAR22AQLoub-vS0IjZBugc24KYBrNL1DZBTUaoVIQCf2SYKKaCKRiEgiy2xQDHAKOAvtKTpgWyJUZnGCMqTWaBQMQAbYkprYApAXxk6tKFhBIDbGFjWzqQSUMT0N1AcMOwQFMghqSxTdsuMUMnQBGbCKxo2bpeSsKCAV0QwAYBDCJVRFItpIMAEHbMlGaxJU9DB3ENE8Nvgii4hjCbghfAmCUKnixZ0r5YuteAa2m8OJhMCEXgIn1lJtElLWbYKDYyA2EsEIMwg2LpXkuGFBihlEQmRoToMmoMickgGJgJEiCbge5JKCeRjwwKZCACQXkAB7snS8cwGEhJmQ16I4vE-pjABNs63Gc6scGSJlihqEWwVjNnexBLj-3ItUnS2nTZc6ByfpSLIb6RvaMJ-OgHbSFTB3haT1ZNvLQhHPSExLt3RP9L1tCkuU77Ym06cFwBtzWYT_2e1o7kLksCj4wsvDwToPn9tHgv8drcW0NbYZNstGvxjpPm9LWZP_4fPvLD_50IrTvg3pOYY2stIVsEd5aQWWF-IV4gP0HOS3Q_WSqfmOdk3T5nD63bRO1BFI3C05OT6XR6POLRwO-LFxD9uOsPT0ZgZciQ5TjBl6UfTirwr1ppVCs3o6n3udG7qla_3n3tv3Ruf_2Mfl18q3rfxNN3MRl993z-3Jz069_Ity-314FzF1Vuyl7tZmZ2KuHtRcXDDdfh9Orqya4N2g8mw9bjM355_HJdNm4-vK_d6rfvp9Nyc_zzsvJsX5gTfDOts8-17sSz7dqnn7Ovz-HlfafzcPnS_Gp_MScWvtWHTvX6M3usXnkdp2xNv-Hg4p483_X61cv37GkURdff8BjX7h5q4s65HA7H37_TD32Lf7Frv16uf_z4ed2wy9VK1zO_2te3ozuGP5anYszvHu-c4LKtX_TubyboZjZ4Md7r18-D4IoOJlcv02-3I_H4IzAs60O9MmAf7x_8ZmeAH3jncvbr5ywg7y-mUfAVX_y6nd3iz9ek6dB694sXmKOGl7ho7pGz5ITZPI6Tq2XEQvGwoNeLkkKabhlvgJNpRAEOygKhKnDUsIgCHMa2rUKPWsy0VehB8lMAs6muYhXInEraQsFT01YmRhU4eQpAAc6gkKlV5ENMST5ZvpX4GshWiRaZ8JXsQm0VrzFMVLgiaDZUuFIGaqjA6VRX0hYaGqLkDV2Jra1bupIaxFYysmxpVJwLXZVKEED_pMQX2ilThS-2qKnCF9vQ6ygtImgvVPSwKGFKiwP6UxV6ukGw0uKwLVuJL1UKU2RipJYiFZMQNHoq-kK_zJTiGdalSrwY0KlaSnZhjCmtD5gc1JKQqbTebGaplSyTKcULjHdK2U83iaEUVzDoKMCZNlNR1zZtpRLDbCWusoNXMQpBm5wBM_my24ELGL_jSV3O5_LDRz9KOlZ5c34RT--PjphqoeBBd9COgvNiJ35_pP3y_eHX82IZ0WMaD7U2DEjITN6z3AgeNfhosbEhgdOXNFb6jibi0YUDLXwQv16abzVIwC8AxKxjBmORDuMa05MDCmfxFka6lyA_t0WyKTEORVt0fa_3WUBD78W3My92JKg24YHD5SYGSd5UpVttLS57-VPt4WP97uGy0HJCIUeLaCC05sAXnvNSaPghd7raBzColrwQLDwmxE61ZiA0dIyO9dVbZPVWbRyAklEheTUPQ8TiCdlEPX1BdaphvZD67FSrluGnEJuwJX6easwo1Icj1-k68iku_JsqeIpejxD7S4t8jbuuttyTT14QyvEmXALDkPl6ZKlCk9d_AzknnerHxuvR31QvM_0frcFfnOF4qMmXsIUMwN-WXka6_s_q6CZfe2lyUEvesM51Xopfwq8spqXF5_REslE7EgGEoPTxFnH-RkYZgzRKmJJJqvYwlV6-ksponeVyZOyGzNoE62Vs_BPDr8MdYWPT_XR-9T3t1p8mRlydW-en1tLJdToQ3gp0jXueH2kdoSUHA3obKMr13_B7LSmpFssrj90byVIg8wyxFYLuhWB7IeanTOONyOW6lI_w6lpsVVqXhY-QcyRA4dr1A0e-NGy64M9CLeB90PFUk-miAIut77ycagsfMGy-Ll7bSdURegtkUPv11u9ytwpQSUhWvBiaLIDlF4kK7XE_i4lel0fcrPWH0v8Xwp2kBBenc9ECMCa5cXG3youf7BK3sktcLxyZWMZPQr9whKiZDcvYz0fU2hh7ywWhob9kgouXSWZpbHYM2eWYauAMwwh00qBI8F641zWUvs5j-VFADQGaxgbnGGzVg1YOn6xA7rV7xuaWvpZWI_4UnsYVby35leUyT44CSYvHhyrme0zJjtYK8ld_rEX8WQD_2AHpplc_8IcJLqzbG2BaOLIMZbeSHYli92pEb1Zj2_XnZ9CSMqplOr_F-bMl0Moxhjkc3gTnR-ESYn5WL63fO2p3tkxvkA61p3ykVTqzMIR6lfZAG5q8TajraFgNbTdUrGhOgTbpskeoqnCjA1GuXB4-a0RJJLxJJJLPvGpGwH8IjeYTkuXjxvJbM0887fHAjXCHIjqU0YZIYnn47Ak_qOQTEeYhTPKvIpSHH80XCyhf5NH97lEx7KFkN4VXTsWV8o6RQ54cOZDsY7MoWwe6b7OVc8WXkQdJaUmqOCKXnkZus-Icfmf5VhLJF77k8Cij-dN-jphGuY1PVAKCqgDlEPs3mh9FT77pUJW9wHJEZX43_Ea4qKZkiau4atRBc1iuJY-64jwpBuc2b850oe7PNxOOgjpEwU4ovxtobmsZeZyjsl7eTHnKQb5HpMpw7G6obcn0CiNjvJEd7w7H38Xyvb7z9OabVXLIdOQZ1_irLmD9otbxfVdwL917fvtlLBhOV78Ctw9BeGI4q4dVPwyXX_eK_ybNW9jh4u-qfHBcVx6FTA6dZL71RTegLdRYx9on2wKx4U-2aH92Mrfd2cn6HwP6P_hbDjk=
```

**Comment by robert-clayton** [comment]
> 
> 
> Ah, I see - any way to manually adjust the values locally so I can use the level 20 values that are in the patch notes? Don't really care too much about having correct values for levels 2-18 and 21+.

https://github.com/PathOfBuildingCommunity/PathOfBuilding/blob/4e9a712673fa915c4d9bf55b1a1dd488ed352836/src/Export/Skills/act_int.txt#L1391

New to looking at the source of this repo, but perhaps changing the `* 0.2` to `* 0.35` here would work?

**Comment by LocalIdentity** [comment]
Fixed in 2.12.0
